help/destinations/catalog/email-marketing/sendgrid.md

```diff
@@ -0,0 +1,201 @@
+| | last_name | String | The contact's family name | maxLength: 50 |
+| | postal_code | String | The contact's ZIP code or other postal code | |
+| | state_province_region | String | The contact's state, province, or region | maxLength: 50|
+| | line | String | The contact's Line account *(if any)* | |
+| | whatsapp | String | The contact's Whatsapp account *(if any)* | |
+| | facebook | String | The contact's Facebook account *(if any)* | |
+| | unique_name | String | | |
+| | phone_number | String | The contact’s phone number | |
+| | custom_fields | Object | Custom field created by the user using the Create Custom Fields Definition API. `This mapping is not supported.` | |
+
+* After completing the mappings, select *Next* to obtain the below review screen indicating the audience and destination are connected.
+
+![](../../assets/catalog/email-marketing/sendgrid/23.jpg)
+
+* Select *Finish* to complete the setup.
+
+![](../../assets/catalog/email-marketing/sendgrid/24.png)
+
+## Validate data within Sendgrid {#validate}
+
+* To validate that you have correctly set up the destination, navigate to the list of destinations. Ensure that the destination you created is visible and the status is enabled.
+
+![](../../assets/catalog/email-marketing/sendgrid/25.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/26.jpg)
+
+* Switch to the *Activation data* tab.
+
+![](../../assets/catalog/email-marketing/sendgrid/27.jpg)
+
+* Monitor the segments to check the flow of data.
+
+![](../../assets/catalog/email-marketing/sendgrid/28.jpg)
+
+* Navigate to the Sendgrid site and check if the new email record from the dataset you created within the [prerequisites](#prerequisites-prerequisites) is being populated within the new contact list.
+
+![](../../assets/catalog/email-marketing/sendgrid/29.jpg)
+
+* Also check a couple of emails to validate if the field mapping is correct.
+
+![](../../assets/catalog/email-marketing/sendgrid/30.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/31.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/32.jpg)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](https://experienceleague.adobe.com/docs/experience-platform/data-governance/home.html).
+
+## Additional resources {#additional-resources}
+
+This Sendgrid destination leverages the below API's
+  * [Sendgrid Marketing Lists](https://api.sendgrid.com/v3/marketing/lists) 
+  * [Sendgrid Marketing Contacts](https://api.sendgrid.com/v3/marketing/contacts)
+* The *Create a List* API is used to create a segment in Sendgrid. 
+* When we execute this API call, we use the value of the *list_name* attribute and concatenate with the current timestamp to create the unique Sendgrid segment.
```

Comment:
two comments here: 

let's please avoid formulations like "when we do something". As specified in a comment further above, we are not there to assist the user. They are on their own, going through the steps. 

Also, more generally, I think that this sentence can be rewritten and moved to the Validate data in Sendgrid section. It is a valuable callout for users to know how unique Sendgrid segment names are generated, and I don't think that we should bury it at the end of the document. 

Rephrase suggestion: Unique segment names in Sendgrid are created by joining the value of the `list_name` attribute and the timestamp of the data export.